src/test/java/petshop/post/FlowPetTest.java

```diff
@@ -34,15 +35,21 @@ public void createPrereq() {
         mypet.setId(133);
         mypet.setName("Animalut");
         mypet.setStatus("available");
+        updatedPet.setId(mypet.getId());
+        updatedPet.setName(mypet.getName());
+        updatedPet.setStatus("sold");
 
     }
 
     @Title("This is a test that will create a new pet, will perform a get method to retrieve it and a put method to update it")
     @Test
-    public void createPet() {
+    public void createGetUpdatePet() {
+
         Response createPetR = postPetSteps.createNewPet(mypet);
-        Pet createdPet = createPetR.as(Pet.class);
-        getPetSteps.getPetWithId(createdPet.getId());
+        commonMethodsPetSteps.statusCodeSuccess(createPetR);
+        commonMethodsPetSteps.getTheIdOfTheCreatedPet(mypet);
+        commonMethodsPetSteps.assertUserRetrievedIsUserCreated(mypet);
+        putPetSteps.updatePet(updatedPet);
```

Comment:
After you update a pet, you should have a method to check the pet was updated. Otherwise this step can fail/not happening and this is not validated.